docs/release_note_generator/main.py

```diff
@@ -0,0 +1,71 @@
+# Usage:
+# GITHUB_TOKEN=<token> python main.py --old <old> --new <new> --path <path>
+#
+# The above command will create release notes for changes between the <old> commit and the <new> commit.
+# Only commits that change files in <path> will be included. A github token is required to avoid rate limits.
+
+import argparse
+import os
+import requests
+
+GITHUB_TOKEN_ENV_VAR = "GITHUB_TOKEN"
+COMMITS_ENDPOINT = "https://api.github.com/repos/dydxprotocol/v4-chain/commits"
+GET_COMMIT_ENDPOINT = "https://api.github.com/repos/dydxprotocol/v4-chain/commits/%s"
+LIST_COMMIT_PULLS_ENDPOINT = "https://api.github.com/repos/dydxprotocol/v4-chain/commits/%s/pulls"
```

Comment:
Now that `v4-chain` includes indexer and js/py client code, do we want to keep track of non-protocol changes too? I am assuming that this will pull all changes including indexer + client changes

Reply:
Got it. Can you add more comments around the path? For example, add an example to use `/protocol` to only grab protocol related commits